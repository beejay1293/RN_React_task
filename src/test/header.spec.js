import React from 'react';
import { shallow } from 'enzyme';
import  Header from '../components/Dashboard/header';

describe('test Header component', () => {
  it('should render component', () => {
    const Head = shallow(<Header  />);
    expect(Head).toMatchSnapshot();
  });
});
