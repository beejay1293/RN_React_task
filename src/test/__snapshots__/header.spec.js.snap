// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Header component should render component 1`] = `
<Fragment>
  <header
    className="header__container"
  >
    <div
      className="logo"
    >
      <h1
        className="header"
      >
        <span>
          <em>
            COIN
          </em>
        </span>
        <b
          className="coin"
        >
          marketcap
        </b>
      </h1>
    </div>
  </header>
</Fragment>
`;
